test/events.js

```diff
@@ -1,92 +1,79 @@
-const getCenterPoint = (xScale, yScale, element) => element.getCenterPoint();
-
-export function testEvents(options, eventIn, eventOut) {
-  testEnterEvent(options, 1, eventIn);
-  testLeaveEvent(options, 1, eventOut);
-  testClickEvent(options, 1, eventIn);
-  testEnterEvent(options, 0, eventOut);
-  testLeaveEvent(options, 0, eventIn);
-  testClickEvent(options, 0, eventOut);
-}
-
-function testEnterEvent(options, toBe, getEventPoint) {
-  const context = getTestCaseContext(toBe);
+export const getCenterPoint = (chart) => window.getAnnotationElements(chart)[0].getCenterPoint();
+export const eventPoint0 = {
+  x: 0,
+  y: 0
+};
 
+export function testEvents(options) {
   describe('events', function() {
```

Comment:
could have the `on on ${options.type}` in this describe statement so the tests would not need it

Reply:
done